internal/operators/ocs/validations.go

```diff
@@ -228,13 +232,16 @@ func (o *operator) setStatusInsufficientResources(nodeInfo *resourceInfo, mode o
 }
 
 // count all disks of drive type ssd or hdd
-func getValidDiskCount(disks []*models.Disk) int {
-	var countDisks int
-
+func getValidDiskCount(disks []*models.Disk) (int64, error) {
+	var countDisks int64
+	var err error
 	for _, disk := range disks {
-		if disk.SizeBytes >= conversions.GbToBytes(MinDiskSize) && (disk.DriveType == ssdDrive || disk.DriveType == hddDrive) {
+		if (disk.DriveType == ssdDrive || disk.DriveType == hddDrive) && !disk.Bootable {
```

Comment:
@omertuc

Reply:
Bootable just indicates that it currently has a bootable partition.  You should check similarly to DiskIsEligible() in internal/hardware/validator.go - size and drive type.  In addition, you should not include the currently selected installation disk.  This can be seen in the installation disk ID property of the host resource (not installation disk path which is deprecated).

Reply:
please use hostutil to fetch the installation disk or the disk identifier (Just a tiny technical appending to the @avishayt  comment)

Reply:
Will the installation disk have been selected already by the AI at this point?

Reply:
Yes

Reply:
Note that the chosen disk may change, so each time the validation is run the currently selected disk should be taken into account